src/Datadog.Trace/DiagnosticListeners/AspNetCoreDiagnostics.cs

```diff
@@ -0,0 +1,67 @@
+#if NETSTANDARD2_0
+
+using System;
+
+namespace Datadog.Trace.DiagnosticListeners
+{
+    /// <summary>
+    /// Instruments ASP.NET Core.
+    /// <para/>
+    /// Unfortunately, ASP.NET Core only uses one <see cref="System.Diagnostics.DiagnosticListener"/> instance
+    /// for everything so we also only create one observer to ensure best performance.
+    /// <para/>Hosting events: https://github.com/aspnet/Hosting/blob/dev/src/Microsoft.AspNetCore.Hosting/Internal/HostingApplicationDiagnostics.cs
```

Comment:
Update link: That repo no longer has a dev branch: https://github.com/aspnet/Hosting/blob/master/src/Microsoft.AspNetCore.Hosting/Internal/HostingApplicationDiagnostics.cs

Reply:
Good catch, thanks.